tools/prowtrans/cmd/prowtrans/main.go

```diff
@@ -806,13 +811,29 @@ func updateExtraRefs(o options, job *config.UtilityConfig) {
 // updateHubs updates the docker hubs for container images
 func updateHubs(o options, job *config.JobBase) {
 	for i := range job.Spec.Containers {
+		tag, _ := dockername.NewTag(job.Spec.Containers[i].Image)
+		repostr := tag.Context().Name()
 		for in, out := range o.HubMap {
-			job.Spec.Containers[i].Image = strings.ReplaceAll(
-				job.Spec.Containers[i].Image,
+			repostr = strings.ReplaceAll(
```

Comment:
nit: I think `baseRef` would make a better name

like https://github.com/google/go-containerregistry/blob/63e6f4ed9c31bfefe26495cc9f3bd1f794174452/cmd/crane/cmd/append.go#L32

"RepositoryStr" (at least in the context of go-containerregistry) seems to only apply to a portion of this string, like https://github.com/google/go-containerregistry/blob/7e30746dde2f5a19b609f885dfef4e7b1d1503ad/pkg/name/repository.go#L51

Reply:
Good point. Fixed.